schema_based_on_diagram.sql

```diff
@@ -0,0 +1,54 @@
+CREATE TABLE patients(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR,
+  date_of_birth DATE,
+  PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_histories(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  admitted_at TIMESTAMP,
+  patient_id INT,
+  status VARCHAR,
+  PRIMARY KEY(id),
+  CONSTRAINT fk_patient_id FOREIGN KEY(patient_id) REFERENCES patients(id)
```

Comment:
- [OPTIONAL] Kindly make sure all FOREIGN KEYS have indexes created. Please follow this example `CREATE INDEX ON medical_histories (patient_id);` and implement the same for all other foreign keys with the appropriate foreign key and table names